packages/user-ui/src/pages/components/institution_box.rs

```diff
@@ -24,8 +24,12 @@ pub fn InstitutionBox(lang: Language, institution: OrganizationSummary) -> Eleme
             div { class: "flex flex-col w-full justify-start items-start pt-[40px] pb-[12px] px-[16px] gap-[48px]",
                 div { class: "flex flex-col w-full gap-[10px]",
                     div { class: "flex flex-row w-full gap-[6px]",
-                        div { class: "font-bold text-[16px] text-[#222222] ", "{institution.name}" }
-                        Auth { width: "24", height: "24" }
+                        div { class: "font-bold text-[16px] text-[#222222] h-[25px] truncate",
+                            "{institution.name}"
+                        }
+                        div { class: "w-[24px] h-[24px]",
```

Comment:
why does this div is required?

Reply:
This is to preserve the size.
If this part is missing, the icon sometimes breaks.
I added this part to maintain the icon size.